www/src/examples/Accordion/AllCollapse.js

```diff
@@ -1,22 +1,34 @@
 <Accordion>
-  <Card>
-    <Card.Header>
-      <Accordion.Toggle as={Button} variant="link" eventKey="0">
-        Click me!
-      </Accordion.Toggle>
-    </Card.Header>
+  <Accordion.Item>
+    <Accordion.Header>
+      <Accordion.Toggle eventKey="0">Accordion Item #1</Accordion.Toggle>
+    </Accordion.Header>
     <Accordion.Collapse eventKey="0">
-      <Card.Body>Hello! I'm the body</Card.Body>
+      <Accordion.Body>
```

Comment:
The verbose component API starts to feel unnecessary now that "accordion" is standalone component. E.g. you are less likely to mix and match the markup, making the need to provide both the body and the Collapse feel a bit excessive?

Perhaps we should combine the collapse and body components? Still expose Collapse as its own component for customization but have the Body component handle it automatically. Worst case someone can drop the single body class on a div if they need to for custom cases

Reply:
Looks like we can do the same for the header too so we can potentially reduce the markup to something like this:

```
<Accordion.Item>
  <Accordion.Header eventKey="0">
    Accordion Item #1
  </Accordion.Header>  
  <Accordion.Body eventKey="0">
    Body text
  </Accordion.Body>
</Accordion.Item>
```